src/prototype_room_my.js

```diff
@@ -242,7 +242,10 @@ Room.prototype.executeRoom = function() {
 
   if (spawns.length === 0) {
     this.reviveRoom();
-  } else if (this.energyCapacityAvailable < 1000) {
+  }
+  // Revive Room if energycapacity or storage don't met minimum
+  else if (this.energyCapacityAvailable < 1000 ||
+  this.storage.store.energy < config.nextRoom.minStorage) {
```

Comment:
In my case i had no storage and a 300 Energy Harvester running around getting nothing done.
Room was set to active.
With the `energyFromStorageThreshold: 2000` it shouldn't trigger revive until needed.